hw/ip/alert_handler/data/alert_handler_env_pkg__params.sv.tpl

```diff
@@ -0,0 +1,6 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+parameter uint NUM_ALERTS = ${n_alerts};
+parameter bit [NUM_ALERTS-1:0] ASYNC_ON = ${async_on};
```

Comment:
i think weicai mentioned during the dv meeting that we can just use design's auto-generated `reg_pkg` instead of creating another package file here. But I remember before we are trying to avoid using any packages from design. Wonder if any other reviewers have any preference?

Reply:
I'm fine with either way.

Reply:
Hi @sriyerg,
Just checking if you have any preference on this? Do you prefer to use design's design's auto-generated `reg_pkg` or prefer to generate a separate one just used by DV?

Reply:
My initial concern for not using the reg pkg for register stuff, is if the designer made a mistake in the reg pkg template. Since we owned the DV specific UVM reg pkg template, that allows us to 'cross check' that the reg pkg / reg top is correct. But as such, the concern is low since the template has been proven with multiple IPs having gone through the DV. I would say, avoid if possible. If not, using reg pkg is probably fine due to low risk. 

Thoughts?

Reply:
I see, thanks for your input Sri. I agree with your concern. Then I will proceed with the current approach - DV alert_handler won't share the same package as design. We will handwritten IP level parameter in our own DV package, and for top_level, we will use the same method to auto-generate the DV package.